src/commands/dev/mod.rs

```diff
@@ -29,6 +29,16 @@ pub fn dev(
     // before serving requests we must first build the Worker
     build_target(&target)?;
 
+    if let Some(user) = &user {
+        hydrate_durable_object_bindings(user, &deployments, &mut target)?;
+    } else {
+        failure::bail!(
```

Comment:
I think I have too much detail in this error message on second thought. Maybe "Binding to a Durable Object namespace is not supported using unauthenticated preview. Please use `wrangler login` or `wrangler config`"